pkg/client/cache/reflector.go

```diff
@@ -227,7 +227,10 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	resyncCh, cleanup := r.resyncChan()
 	defer cleanup()
 
-	options := unversioned.ListOptions{}
+	// Explicitly set "0" as resource version, as we can handle some
+	// delays in the reflector framework (those will be caught by
```

Comment:
I'm not sure I understand the comment correctly:
// Explicitly set "0" as resource version, as we can handle some
// delays in the reflector framework and all changes will be caught by
// the watch eventually.